components/form/LocationField.tsx

```diff
@@ -73,35 +93,25 @@ export const LocationField = React.forwardRef<RNView, LocationFieldProps>(({name
       fillOpacity: feature.properties.fillOpacity,
     })) ?? [];
 
-  const onPress = useCallback(
-    (event: GestureResponderEvent) => {
-      void (async () => {
-        const point = {x: event.nativeEvent.locationX, y: event.nativeEvent.locationY};
-        const coordinate = await mapRef.current?.coordinateForPoint(point);
-        if (coordinate) {
-          field.onChange(latLngToLocationPoint(coordinate));
-        }
-      })();
-    },
-    [field],
-  );
   const emptyHandler = useCallback(() => undefined, []);
 
   return (
-    <VStack width="100%" space={4} ref={ref}>
-      <BodySmBlack>{label}</BodySmBlack>
-      <TouchableOpacity onPress={toggleModal} disabled={disabled}>
-        <HStack borderWidth={2} borderColor={colorLookup('border.base')} borderRadius={4} justifyContent="space-between" alignItems="stretch">
-          <View p={8}>
-            <Body>{value ? `${value.lat.toFixed(5)}, ${value.lng.toFixed(5)}` : 'Select a location'}</Body>
-          </View>
-          <Center px={8} borderLeftWidth={2} borderColor={colorLookup('border.base')}>
-            <FontAwesome name="map-marker" color={colorLookup('text')} size={bodySize} />
-          </Center>
-        </HStack>
-      </TouchableOpacity>
-      {/* TODO: animate the appearance/disappearance of the error string */}
-      {error && <BodyXSm color={colorLookup('error.900')}>{error.message}</BodyXSm>}
+    <>
+      <VStack width="100%" space={4} ref={ref}>
```

Comment:
I added this fragment to pull out the modal code from the `VStack`. It doesn't seem to change the functionality at all, but I think it makes it easier to read and understand what's going on